appcues/src/main/java/com/appcues/util/StringExt.kt

```diff
@@ -9,3 +9,84 @@ internal fun String.toSlug() = lowercase(Locale.getDefault())
     .joinToString("-")
     .replace("-+".toRegex(), "-")
     .trimEnd('-')
+
+@Suppress("TooGenericExceptionCaught", "SwallowedException")
+internal fun String.beautify(indentationMultiplier: Int): String {
```

Comment:
Extracted the beautify method so it could be reused in the debugger 
+ healthy code organization